app/src/main/java/openfoodfacts/github/scrachx/openfood/features/product/view/summary/SummaryProductFragment.kt

```diff
@@ -462,6 +474,44 @@ class SummaryProductFragment : BaseFragment(), ISummaryProductPresenter.View {
         }
     }
 
+    private fun showListChips() {
+
+        // remove the existing childviews on chip group if any
+        binding.listChips.removeAllViews()
+
+        val listDao: YourListedProductDao = OFFApplication.daoSession.yourListedProductDao
+
+        //Query to check if the product belons in any of thr user's list
+        val queryResult : List<YourListedProduct> = listDao.queryBuilder()
+                .where(YourListedProductDao.Properties.Barcode.eq(product.code)).list()
+
+        if (queryResult.isNullOrEmpty()) {
+            binding.listChips.visibility = View.GONE
+            return
+        }
+
+        binding.listChips.visibility = View.VISIBLE
```

Comment:
You could add an else branch to the precedent statement.

Reply:
Ok. I will do that. Thanks.

Reply:
> I don't like the gray, could we use a "more white" white or even a light gray

![image](https://user-images.githubusercontent.com/71206228/105214275-ccc9a380-5b75-11eb-8e5d-2f17724ba594.png)

@teolemon @VaiTon Is this fine?